File: week05/QtAutomata/Model/afnd.cpp
#include "afnd.h"

AFND::AFND()
{

}


void AFND::addState(State _state){
    std::cout << "Adding state into the AFND in the model with label:" << _state.getLabel().toStdString()
              << "and value " << _state.getValue() << std::endl;
}

File: week05/GUIAutomata/src/model/AFND.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

/**
 *
 * @author christian
 */
public class AFND implements IAutomata {
    private State state;
    
    @Override
    public void addState(State state){
        this.state = state;
    }
}
